src/Form/RenderBuildOptionsTab.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+
+import { SplitSchema, Step } from '../../types/interfaces'
+import { setStepState } from '../../utils/formUtils'
+
+import Box from '@mui/material/Box'
+import Stack from '@mui/material/Stack'
+import FormGroup from '@mui/material/FormGroup'
+import FormControlLabel from '@mui/material/FormControlLabel'
+import Checkbox from '@mui/material/Checkbox'
+import Divider from '@mui/material/Divider'
+import Typography from '@mui/material/Typography'
+import FileInput from '../common/FileInput'
+
+export function RenderBuildOptionsTab({
+  currentStep: step,
+  splitSchema,
+  setSplitSchema,
+}: {
+  currentStep: Step
+  splitSchema: SplitSchema
+  setSplitSchema: Function
+}) {
+  const { state } = step
+  const { rawModelExport, allowGuestDeployments } = state
+
+  const handleRawModelExportChange = (event: any) => {
+    setStepState(splitSchema, setSplitSchema, step, { ...state, rawModelExport: event.target.checked })
+  }
+
+  const handleGuestDeploymentChange = (event: any) => {
+    setStepState(splitSchema, setSplitSchema, step, { ...state, allowGuestDeployments: event.target.checked })
+  }
+
+  return (
+    <Stack direction='column' spacing={2} sx={{ p: 3 }}>
```

Comment:
This should be within the jsonschema and use the existing form instead of creating a new one.  Custom tabs should only be used for things that _can't_ be done within the schema, like the submission tab (no data) and the file upload tab.